src/modules/mockFunctions/editTask.js

```diff
@@ -0,0 +1,16 @@
+// const editTask = (array, description, index) => {
+//   const updatedTask = [...array];
+//   updatedTask[index] = { ...updatedTask[index], description };
+//   return updatedTask;
+// };
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: